dotCMS/src/main/java/com/dotcms/rendering/velocity/services/DotResourceLoader.java

```diff
@@ -74,6 +74,7 @@ public InputStream getResourceStream(final String filePath) throws ResourceNotFo
                 }
 
             } catch (Exception e) {
+                Logger.error(this, e.getMessage(), e);
```

Comment:
Can you improve the error message please? Add info such as the `filePath`

Reply:
Done